webpack.config.js

```diff
@@ -190,41 +196,19 @@ module.exports = (env, mode) => {
                         'package.json'
```

Comment:
#Compared to the one from solid it seems it miss definitions.cnd and import.xml

Reply:
They are actually here, both.